doc/DEMO.md

```diff
@@ -245,6 +252,47 @@ cargo build
 
 Now we need to setup `revaultd` three times, one for each entity.
 
+#### Generating the Miniscript descriptors
+All wallets are going to track the same coins, which are specified using the [output descriptors language](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md).
+
+Since we want all wallets to be using the same descriptors, we are going to generate them once and for all.
+For this purpose you can use the `mscompiler` tool:
+```
+cd contrib/tools/mscompiler
+cargo build
+```
+
+This small command line tool takes 6 parameters as input, allowing to tweak the configuration of the
+Revault setup at the chain level. These tweakable parameters are:
+- The xpubs of the stakeholders
+- The raw public keys of the cosigning servers (as many as stakeholders xpubs)
+- The xpubs of the managers, and the threshold of the multisig (eg you may have 3 xpubs and a
+  threshold of 2 in order to allow 2 managers to Spend while the third one is on vacation)
+- The xpubs of the managers' "CPFP wallet". This functionality isn't implemented yet so you can put
+  garbage for now but it will eventually allow managers to speed up the confirmation of a Spend
+  transaction during a high-fee period.
+- The value of the Unvault relative timelock. It specifies how much time do the stakeholders and all
+  their watchtowers have to Revault a spending attempt from the managers should they feel like it.
+
+Use the xpubs you previously generated, and make the other parameters vary as you'd like. Here is an
+example of usage:
+```
+./target/debug/mscompiler '["xpub6CFH8m3bnUFXWXxKVQjMXqMiQWYRhcTeZCW1QghmkNeGkPFwADfFNt9JMuW38MnYVSAV9eyqJ3A61kbsfC5PSCdkZWi7pD2L4jv6edaPxKp","xpub6FEZyiJxqwu7zkqqVGXVbGhcAj1L5imn55VVa4Mk5WE46xdAKgD4uSR9ems9EehAApZPVXFrxcLQ7zPqYywu1z4Cjhesyp7HeRSgSdUq1BB","xpub6CFH8m3bnUFXa6UB8KhTMTPDz3cNQ9wAsQ9fGM52WZ1jBenPtf1GED6fJoDmpEYQQkk3VUHFN5ZRDLV7SRgX4M8KMTpRTH9zGRzg5udqwwo","xpub6ETaaosT68a6mxPp1dRh1yeGjMZonsZMxA1SA95iqKWDcQUXxQPYFyottUR58E8qjjnAwPcEtYS9iejkERbnGuNqfF2wgToLcxzf97FHevs"]' '["02644cf9e2b78feb0a751e50502f530a4cbd0bbda3020779605391e71654dd66c2","03ced55d1208bd8c6b42b11e29baa577711cae831b3a1296607c5e5d3ed365f49c","026237f655f3bf45fd6b7aa00e91c2603d6155f1cc001e40f5e47662d965c4c779","030a3cbcfbfdf7122fe7fa830354c956ea6595f2dbde23286f03bc1ec0c1685ca3"]' '["xpub6CFH8m3bnUFXvS78XZyCQ9mCbp7XmKXbS67YHGUS3NxHSLhAMCGHGaEPojcoYt5PYnocyuScAM5xuDzf4BqFQt3fhmKEaRgmVzDcAR46Byh","xpub6ECZqYNQzHkveSWmsGh6XSL8wMGXRtoZ5hkbWXwRSVEyEsKADe34dbdnMob1ZjUpd4TD7no1isnnvpQq9DchFes5DnHJ7JupSntZsKr7VbQ"]' 2 '["xpub6BhQvtXJmw6hi2ALFeWMi9m7G8rGterJnMTNRqUm29uvB6dVTELvnEs7hfxyN3JM48FR2oh4t8chsvw7bRRRukkyhqp9WZD4oB9UvxAMpqC","xpub6BhQvtXJmw6hksh9rRRfdLjaWjQiNMZWtkM5ebn8QkAgh5na2Un6mCDABwkUmHhPCMYtsM9zHY5jxbQ86ayvjfY8XtavbovB6NcNy8KyQLa"]' 18
```

Comment:
We've been using `cargo run` for the whole tutorial, I would use it as well too (or at least modify the other points where it was used)

Reply:
Also, this is quite complicated to read as it's all displayed on a single row. Can you either
1. Somehow split it into multiple rows or
2. Put some placeholders instead of the actual xpubs, something like:
`'["first_stakeholder_xpub", "second_stakeholder_xpub"]'` etc

Reply:
I think it's valuable as a "real" usage example, what to put is already detailed before and even given by the tool:
```
Usage: '/home/darosior/projects/revault/d/contrib/tools/mscompiler/target/debug/mscompiler <stakeholders xpubs> <cosigs pubkeys> <managers xpubs> <managers threshold> <cpfp xpubs> <Unvault CSV>'
 All values are as JSON.
```
So either we keep it as such or we get rid of it imo.

Updated to use `cargo run`..